backend/src/Routes/elections.routes.js

```diff
@@ -67,10 +69,9 @@ router.get('/Elections',
     authController.getUser,
     electionController.getElections)
 router.post('/Elections/',
-    authController.getUser,
-    authController.isLoggedIn,
-    electionController.createElection,
```

Comment:
Should createElection() be removed from electionController now that it's implemented by createElectionController.createElectionController?

Reply:
Yes!  Great catch, thanks

Reply:
Thanks for the update, looks good!